tests/unit/amo/components/TestAddonReviewCard.js

```diff
@@ -562,11 +569,12 @@ describe(__filename, () => {
   });
 
   it('lets you flag a developer reply', () => {
-    const { reply } = _setReviewReply();
-    const root = renderReply({ reply });
+    const review = createReviewAndSignInAsUnrelatedUser();
```

Comment:
This test was giving me some trouble - not sure if this is the route for us to take. Feedback greatly appreciated!

Reply:
> This test was checking a developer's reply to an existing review. I think I changed the logic behind it - would appreciate feedback.

Why did you need to change the test? I put it back to how it was and it passed for me. Maybe it was related to not being signed in?